SystemInterface/Diagnostics/IProcess.cs

```diff
@@ -73,6 +78,27 @@ public interface IProcess
         [MonitoringDescription("ProcessStandardOutput"), DesignerSerializationVisibility(DesignerSerializationVisibility.Hidden), Browsable(false)]
         IStreamReader StandardOutput { get; }
 
+        /// <summary>
+        ///  Gets a new System.Diagnostics.Process component and associates it with the currently active process.
+        /// </summary>
+        /// <returns>A new System.Diagnostics.Process component associated with the process resource that is running the calling application.</returns>
+        Process GetCurrentProcess();
```

Comment:
Changed approach. I introduced IProcessFactory that is responsible for creating IProcess object. So basically all Process static methods are placed there.
Now we are able to return IProcess instead of Process